modules/46_carbonpriceRegi/NDC/declarations.gms

```diff
@@ -20,6 +20,8 @@ p46_vm_CO2eq_2020(all_regi)                              "2020 reference emissio
 p46_taxCO2eq_iter(iteration,ttot,all_regi)               "CO2eq tax non-regi tracked over iterations [T$/GtC]"
 p46_taxCO2eqRegi_iter(iteration,ttot,all_regi)           "CO2eq tax regi tracked over iterations [T$/GtC]"
 p46_taxCO2eqLast(tall,all_regi)                          "general carbon price in last iteration [T$/GtC]"
```

Comment:
could you change this declaration from tall to ttot? it is only ever called for "t" or "ttot" in the model, and all other prices are only defined over ttot.